xml/admin_ceph_gateway.xml

```diff
@@ -3234,7 +3239,35 @@ rgw_frontends="beast port=80"
 rgw_zone=us-east-2</screen>
 <screen>&prompt.cephuser;sudo systemctl start ceph-radosgw@rgw.us-east-2</screen>
    </sect3>
-  </sect2>
+ <sect3 xml:id="ceph-update-dashboard-config">
+   <title>Update Dashboard Configuration</title>
+   <procedure>
+     <step>
+       <para>Set the following to update your dashboard configuration for
+       multi-site:</para>
+       <screen>&prompt.cephuser; ceph dashboard set-rgw-api-access-key ACCESS-KEY
+       &prompt.cephuser; ceph dashboard set-rgw-api-secret-key SECRET-KEY
+      </screen>
+      <para>Based on the secret keys and access key variables.</para>
+     </step>
+     <step>
+       <para>Set the dashboard default to <literal>admin</literal>:</para>
+       <screen>&prompt.cephuser; ceph dashboard set-rgw-api-user-id USER-ID</screen>
```

Comment:
We want to deaful to the 'admin' user, right? Then i would supply the 'admin' USER-ID:
```suggestion
       <screen>&prompt.cephuser; ceph dashboard set-rgw-api-user-id admin</screen>
```